pkgs/servers/caddy/default.nix

```diff
@@ -15,6 +23,10 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-NomgHqIiugSISbEtvIbJDn5GRn6Dn72adLPkAvLbUQU=";
 
+  postInstall = ''
+    install -Dm644 ${dist}/init/caddy.service ${dist}/init/caddy-api.service -t $out/lib/systemd/system
```

Comment:
https://github.com/caddyserver/dist/tree/master/init#choosing-a-service-file
I think we should use caddy.service here right ? since we are configuring with a file, right ?

Reply:
Yes. We don't activate the `caddy-api.service` file, just include it in case someone wants it I guess. It won't do much just by being included in the directory.

Reply:
Should we not include the `caddy.service` file though rather than the `caddy-api.service` ?

Reply:
We should definitely include the `caddy.service` file. We don't actually use the `caddy-api.service` so it is optional.

Reply:
oh I misread that line completely, didn't see that you are including both files (it's a little late here). Sorry for the useless comment.